<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RERoomCategoryRateRow</class>
 <widget class="QDialog" name="RERoomCategoryRateRow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Category rate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbCode">
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="EQLineEdit" name="leCode">
       <property name="PrimaryKey" stdset="0">
        <bool>true</bool>
       </property>
       <property name="Field" stdset="0">
        <string>f_id</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Rate</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="EQLineEdit" name="leRate">
       <property name="Field" stdset="0">
        <string>f_rate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Starting</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Bed</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ending</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="EQLineEdit" name="leMonth1">
       <property name="Field" stdset="0">
        <string>f_month1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="EQLineEdit" name="leDay1">
       <property name="Field" stdset="0">
        <string>f_day1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="EQLineEdit" name="leMonth2">
       <property name="Field" stdset="0">
        <string>f_month2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="EQLineEdit" name="leBedCode">
       <property name="Field" stdset="0">
        <string>f_bed</string>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="EQLineEdit" name="leBedName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="EQLineEdit" name="leDay2">
       <property name="Field" stdset="0">
        <string>f_day2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>57</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnOk">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>56</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbCategory">
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="EQLineEdit" name="leCategory">
       <property name="Field" stdset="0">
        <string>f_category</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leMonth1</tabstop>
  <tabstop>leDay1</tabstop>
  <tabstop>leMonth2</tabstop>
  <tabstop>leDay2</tabstop>
  <tabstop>leBedCode</tabstop>
  <tabstop>leBedName</tabstop>
  <tabstop>leRate</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>leCode</tabstop>
  <tabstop>leCategory</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
